cms_form_compassion/forms/match_partner_form.py

```diff
@@ -122,6 +125,7 @@ def form_before_create_or_update(self, values, extra_values):
                 partner.write(source_vals)
             self.partner_id = partner.id
             values['partner_id'] = partner.id
+            partner.active = True
```

Comment:
Instead of doing another write, you could use the write at line 125 to put it active here.